multihost_test/Bugzillas/test_bz_automation.py

```diff
@@ -120,3 +120,25 @@ def test_bz672510(self, multihost):
         execute_cmd(multihost, f"sh /tmp/bz672510_3.sh {tuser} {tgroup}")
         execute_cmd(multihost, f"userdel -rf {tuser}")
         execute_cmd(multihost, f"groupdel  {tgroup}")
+
+    def test_bz787736(self, multihost, create_backup):
+        """
+        :title: bz787736-pwconv-grpconv-skips-2nd-of-consecutive-failures
+        :id: 017f615a-92a8-11eb-bca1-002b677efe14
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=787736
+        :steps:
+          1. If /etc/shadow (or /etc/gshadow) contains consecutive
+           bad lines pwconv only fixes the first, skipping the 2nd.
+        :expectedresults:
+          1. Should not succeed
+        """
+        execute_cmd(multihost, "echo 'example1:!!:15372:0:99999:7:::' >> /etc/shadow")
+        execute_cmd(multihost, "echo 'example2:!!:15372:0:99999:7:::' >> /etc/shadow")
+        # pwck first try should end with 2
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "pwck -r > /tmp/anuj")
```

Comment:
Where are you checking that `pwck` returns 2?

Reply:
Here 2 means failure  :                  with pytest.raises(subprocess.CalledProcessError):

Reply:
Thanks for clarifying.